### PR TITLE
Prism-Bukkit-159 pste.me references still show in config

### LINKED ISSUE #130: config.yml and hikari.properties have mismatched values
The defaults in config.yml and hikari.properties have different values.  They should be changed to match, or if the default values in config.yml no longer apply, they should be removed from the code.

In particular, max-pool-connections (20) / maximumPoolSize (10) and min-idle-connections (5) / minimumIdle (10) don't match.  There may be others.

Prism 2.1.6-SNAPSHOT-118
MySQL 5.7.29
git-Spigot-6de3d4b-fc24934 (MC: 1.15.2)

The following were generated for the default files.

config.yml:
```
  mysql:
    hostname: 127.0.0.1
    username: root
    password: ''
    databaseName: minecraft
    prefix: prism_
    port: '3306'
    useNonStandardSql: true
    database:
      max-pool-connections: 20
      min-idle-connections: 5
      max-wait: 30000
      max-failures-before-wait: 5
      actions-per-insert-batch: 300
      force-write-queue-on-shutdown: true
```
hikari.properties:
```
#Prism Hikari Datasource Properties for advanced database Configuration
#Thu Mar 12 11:11:51 EDT 2020
allowPoolSuspension=false
poolName=prism
registerMbeans=false
maxLifetime=1800000
autoCommit=true
leakDetectionThreshold=0
maximumPoolSize=10
idleTimeout=600000
validationTimeout=5000
minimumIdle=10
isolateInternalQueries=false
initializationFailTimeout=1
readOnly=false
connectionTimeout=30000
```

### LINKED ISSUE #159: pste.me references still show in config
Prism 2.1.6 still has references to pste.me in the default configuration file:

```
  paste:
    enable: false
    username: Username on http://pste.me/#/signup
    api-key: API key from http://pste.me/#/account
```

### PR DESCRIPTION
**Pull request by Narimm**
* Improves Config loading
* Ensure hikari can save on first run
* Removes old config values
* Removes a lot of log cruft on first db run
Fixes #159
Fixes #130

Signed-off-by: Narimm <benjicharlton@gmail.com>

### CODE REVIEW

File: src/main/java/me/botsko/prism/ConfigBase.java [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.util.logging.Logger;
```
Comment (Narimm) [inline src/main/java/me/botsko/prism/ConfigBase.java:13]: Codacy Issue found: [Avoid unused imports such as 'java.util.logging.Logger'](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5472183)

**Comment by Narimm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 3
           

Complexity increasing per file
==============================
- src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5472183&bid=17935010)

File: src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java [hunk 2]
```diff
@@ -106,6 +105,11 @@ private static void setupDefaultProperties(@Nonnull ConfigurationSection section
                 }
             }
             try {
+                if (!propFile.getParentFile().exists()) {
+                    if (!propFile.getParentFile().mkdirs()) {
```
Comment (Narimm) [inline src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java:109]: Codacy Issue found: [These nested if statements could be combined](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5472183)

File: src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java [hunk 3]
```diff
@@ -14,9 +15,7 @@
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.OutputStream;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
+import java.sql.*;
```
Comment (Narimm) [inline src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java:18]: Codacy Issue found: [Using the '.*' form of import should be avoided - java.sql.*.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5472183)

**Review by alchemistmatt — DISMISSED** [review]
Looks good to me.

**Comment by Narimm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           

Complexity increasing per file
==============================
- src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5472183&bid=17935010)

File: src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java [hunk 4]
```diff
@@ -106,6 +109,9 @@ private static void setupDefaultProperties(@Nonnull ConfigurationSection section
                 }
             }
             try {
+                if (!propFile.getParentFile().exists() && !propFile.getParentFile().mkdirs()) {
+                        Prism.log("Prism Directory couldn't be created");
```
Comment (Narimm) [inline src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java:113]: Codacy Issue found: ['if' child has incorrect indentation level 24, expected level should be 20.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5472183)

**Comment by Narimm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5472183&bid=17935010)

**Comment by Narimm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5472183&bid=17935010)

**Pull request merged**